components/classes/content-asset.schema.json

```diff
@@ -40,6 +40,11 @@
           "title": "Asset Type",
           "type": "string",
           "description": "Asset type such as image or video."
+        },
+        "xdm:assetThumbnailURL": {
+          "title": "Asset thumbnail URL",
+          "type": "string",
```

Comment:
Seems like a URL, so do you want to add "format": "uri" or "format": "uri-reference" qualifier to this field?